### ISSUE TITLE
Profile | Notification Settings | Update documentation on adding contact information

### ISSUE THREAD

**Issue by ericasauv09** [department-of-veterans-affairs/va.gov-team#48952]
## Background

Once we have implemented the front end changes to go along with the new edit as a subtask pattern we need to update any documentation that still references the old designs or workflows.

## Tasks
- [x] Update [use case documentation](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/notification-preferences/use-cases/partial-or-missing-contact-info.md)
- [x] Update contact center guide ([I believe this was completed in August 2023](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63198))
- [x] Update product outline
- [x] Update any other places this may be referenced
  

## Associated Tickets
- #48410

**Comment by andaleliz** [comment]
Note for self: this is on hold until we learn more on 12/19.

**Comment by andaleliz** [comment]
@BerniXiongA6 do we have a way of tracking tickets anyone on the team can work on? I was previously assigned to this, and added Florence thinking "any designer can do this" and then realized this is a documentation update and anyone on the team can do it, regardless of their practice area.

**Comment by BerniXiongA6** [comment]
Hi @andaleliz -- until our new PM starts, I can compile a list of tickets that could be worked on when team members have downtime. When the PM is here, let's revisit that conversation about how to track these tickets. Tracking future work (not already in the roadmap) is one of the topics we have started to discuss in Product/Delivery, and will prioritize as a team in coming months. In the meantime, feel free to keep the ticket unassigned and we can just make sure the appropriate labels are added. I'll tentatively place this on the sprint planning outline for Sprint 9.

**Comment by andaleliz** [comment]
@mtcA6 FYI changing the assignee to you since anyone on the team can work on this. I fear if it's totally unassigned it could get forgotten but figure that it can be assigned to anyone with bandwidth to pick it up once we're ready to update the documentation.

**Comment by mtcA6** [comment]
@andaleliz that works for me :)

**Comment by mtcA6** [comment]
@andaleliz I'm okay with the reassignment but I didn't notice the pipeline change. Lets keep this in the backlog, it's blocked by the dependency and we haven't even started the dependency. The blocked column should be for things we've picked up in this sprint or a prior sprint that's stuck and we're actively working toward. While we are working on this body of work, we aren't really working on this ticket. That said, I don't want to gum up the blocked column.

**Comment by mtcA6** [comment]
@andaleliz mind taking a look at the [use case documentation ](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/notification-preferences/use-cases/partial-or-missing-contact-info.md)I added?

**Comment by andaleliz** [comment]
I don't mind at all!  Thanks for doing this. 

It makes sense as you've written it, though IMO it should either:
- be renamed to remove "missing contact info" OR
- include the information on the existing missing contact info use case doc so we only have one use case doc that references both.  

I think either way would work but it's a little unclear to have two files that both reference missing contact info.

The only thing I noticed is that in most cases, the mock-up links should probably point specifically to the specific part of the Figma page. I didn't check all of them, but the first one took me to the overall notification settings mock-ups.   I think it works for the sub-task flow to go to the overall file.

To get a link to a specific frame is kind of a PITA and I'm not sure if you have permissions to do it, but give it this a try. If it doesn't work, Pam or I can update the links for ya.

1. Click on the page you want to link to (called a "frame" in Figma, I believe). This will select it in the left panel list of frames/layers
2. Right click on the selection in the left panel
3. Choose `Copy/Paste as`
4. Choose 'Copy link`

<img width="738" alt="image" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/55992116/7b3ef257-551a-4dc9-ba94-19b97648b752">

**Comment by andaleliz** [comment]
Oh! On a Mac there is a keyboard shortcut `⌘ L`! I think the Windows equivalent would be `Ctrl+L`

**Comment by mtcA6** [comment]
Ok, renamed: _Notification settings: User has partial contact information_

Updated the links I think, let me know if that works. When Pam finishes migrating the notification settings page I'll double check to make sure the use case links still work.

**Comment by andaleliz** [comment]
Oh, I meant the URL should be renamed - sorry not to be clear. I renamed it so it's very clear when they're side by side in the use case directory. https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/profile/notification-preferences/use-cases/user-partial-contact-info.md

The links did work! Yay!

**Comment by mtcA6** [comment]
Oh ohhhh ok cool, closing this out! Thanks for the help
